### ISSUE TITLE
Feature Request: Add hyperlink to Changelog.md as part of notifications

### ISSUE THREAD

**Issue by ChrisPearce** [sibbell/sibbell-issues#49]
Often projects lack dedicated release notes being included in the tagged release, and instead rely on developers reading the contents of the Changelog.md file.

It would be really helpful if the email notifications could include a hyperlink to this file (when it exists in the repository). This would enable users to quickly access/view the file straight from the email, rather than having to navigate to GitHub, then find the file and then open it.

**Comment by ykorman** [comment]
+1 This would be really helpful.

**Comment by andrefgneves** [comment]
Yeah, this would be very useful. Better yet would be to include the relevant part of the Changelog file if the release has no notes.

**Comment by davegaeddert** [comment]
@andrefgneves we do try to grab content from the changelog file that way -- it always can use a little tweaking though so if you can point me towards the package + repo + release then I can see if there's a reason we didn't find it.

**Comment by andrefgneves** [comment]
@davegaeddert https://github.com/moxystudio/webpack-sane-compiler-reporter for example.
I get [this](http://d.pr/i/DfMySt) from sibbell. We use [conventional-changelog](https://github.com/conventional-changelog/conventional-changelog) to generate the changelog.

**Comment by davegaeddert** [comment]
Whoops, sorry my bad. I thought this was an issue on https://github.com/dependencies-io/support (our other service). Sibbell does not yet use the changelog (as you suspected) but I think we're going to find a way to merge at least *some* functionality of dependencies.io into sibbell, which would mean that sibbell could start showing CHANGELOG content along with a bunch of other things we're doing with dependencies.io.

**Comment by andrefgneves** [comment]
Awesome, good to know 👍
